pkg/controllers/repositoryserver/server.go

```diff
@@ -48,35 +48,43 @@ func (h *RepoServerHandler) startRepoProxyServer(ctx context.Context) (err error
 		return nil
 	}
 
-	cmd := command.ServerStart(
-		command.ServerStartCommandArgs{
-			CommandArgs: &command.CommandArgs{
-				RepoPassword:   "",
-				ConfigFilePath: command.DefaultConfigFilePath,
-				LogDirectory:   command.DefaultLogDirectory,
+	maxRetries := 5
+	for retries := 1; retries <= maxRetries; retries++ {
+		cmd := command.ServerStart(
+			command.ServerStartCommandArgs{
+				CommandArgs: &command.CommandArgs{
+					RepoPassword:   "",
+					ConfigFilePath: command.DefaultConfigFilePath,
+					LogDirectory:   command.DefaultLogDirectory,
+				},
+				ServerAddress:    repoServerAddress,
+				TLSCertFile:      tlsCertPath,
+				TLSKeyFile:       tlsKeyPath,
+				ServerUsername:   serverAdminUserName,
+				ServerPassword:   serverAdminPassword,
+				AutoGenerateCert: false,
+				Background:       true,
 			},
```

Comment:
Can we move command creation out of the loop, since it wont change on each iteration?